react/features/base/participants/functions.js

```diff
@@ -478,8 +482,14 @@ export function shouldRenderParticipantVideo(stateful: Object | Function, id: st
     }
 
     /* First check if we have an unmuted video track. */
-    const videoTrack
-        = getTrackByMediaTypeAndParticipant(state['features/base/tracks'], MEDIA_TYPE.VIDEO, id);
+    const tracks = state['features/base/tracks'];
+    let videoTrack;
+
+    if (getMultipleVideoSupportFeatureFlag(state) && participant.isVirtualScreenshareParticipant) {
```

Comment:
can we add a helper method for that? it is the same code here https://github.com/jitsi/jitsi-meet/pull/11529/files#diff-d5a1849ae9f1571fe8ac3f9112be492051efb09817de169fff6c700a7a2527a5R258